src/api/app/models/event/service_fail.rb

```diff
@@ -21,6 +21,31 @@ def set_payload(attribs, keys)
       attribs['error'] = attribs['error'][0..800]
       super(attribs, keys)
     end
+
+    def metric_tags
+      error = case payload['error']
+              when /cannot be expanded/
+                700
+              when /bad link/
```

Comment:
or for bad link we could maybe go for status 409 which indicated a conflict

Reply:
Hmm. Wouldn't a conflict cause a "cannot be expanded" error? Then we could use 409 there.